code/ui/ui_shared.h

```diff
@@ -394,23 +394,27 @@ typedef struct {
 	void (*drawCinematic)(int handle, float x, float y, float w, float h);
 	void (*runCinematicFrame)(int handle);
 
-  float			yscale;
-  float			xscale;
+	float			yscale;
+	float			xscale;
   float			bias;
   int				realTime;
   int				frameTime;
-	int				cursorx;
-	int				cursory;
+	float			cursorx;
+	float			cursory;
 	qboolean	debug;
 
   cachedAssets_t Assets;
 
 	glconfig_t glconfig;
 	qhandle_t	whiteShader;
-  qhandle_t gradientImage;
-  qhandle_t cursor;
+	qhandle_t gradientImage;
+	qhandle_t cursor;
 	float FPS;
 
+	float		screenWidth;
+	float		screenXFactor;
+	float		screenXFactorInv;
+
```

Comment:
I declared those here because that's where Ent had declared widthRatioCoef in jaMME, which is what I referenced when writing the widescreen correction in jaPRO.

I'll move it to uiInfo_t